### DISCUSSION TITLE
Upgrade Nuxt 2.15.8 to Vue 2.7.x

### DISCUSSION THREAD

**Discussion by staaph** [nuxt/nuxt#10582] [Questions (Nuxt 2)]
Hello together,

did anyone successfully upgrade the Vue version to 2.7 ?
Im in a Nuxt 2.15.8 project using Vue 2.6.10 and when trying to bump the version i get a fatal error.
<img width="789" alt="Bildschirmfoto 2022-07-18 um 16 10 56" src="https://user-images.githubusercontent.com/101838784/179537073-6a3cc226-8921-42b6-9270-25cfb1ca7a58.png">

Any help appreciated :)
BR
staaph

**Comment by wesbragagt** [comment]
I'm very curious as well and in the same situation.

**Comment by imfaisalkh** [comment]
@staaph You also have to match the server renderer version with vue; run this command `npm i vue-server-renderer@2.7.7 --save-dev`

**Comment by starkfire** [comment]
You may also use `overrides` on `package.json` to make sure their versions match whenever an `npm install` is being performed based on this [comment](https://github.com/nuxt/nuxt/issues/10305#issuecomment-1200902785).

```json
{
    "overrides": {
        "vue": "2.7.14",
        "vue-server-renderer": "2.7.14"
    }
}
```
